index.html

```diff
@@ -12,32 +12,207 @@
+                    </ul>
+                </div>
+                <button class="button card-button rmv" type="button">See Project</button>
+            </li>
+            <li class="project-card bank">
+                <div class="text-container">
+                    <h3 class="project-title">Profesional Art Printing Data</h3>
+                    <p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="list-container">
+                        <li class="project-list">html</li>
+                        <li class="project-list">bootstrap</li>
+                        <li class="project-list">ruby</li>
+                    </ul>
+                </div>
+                <button class="button card-button rmv" type="button">See Project</button>
+            </li>
+            <li class="project-card kards">
+                <div class="text-container ">
+                    <h3 class="project-title">Profesional Art Printing Data</h3>
+                    <p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="list-container">
+                        <li class="project-list">html</li>
+                        <li class="project-list">bootstrap</li>
+                        <li class="project-list">ruby</li>
+                    </ul>
+
+                </div>
+                <button class="button card-button rmv" type="button">See Project</button>
+            </li>
+            <li class="project-card banker">
+                <div class="text-container">
+                    <h3 class="project-title">Profesional Art Printing Data</h3>
+                    <p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="list-container">
+                        <li class="project-list">html</li>
+                        <li class="project-list">bootstrap</li>
+                        <li class="project-list">ruby</li>
+                    </ul>
+                </div>
+                <button class="button card-button rmv" type="button">See Project</button>
+            </li>
+            <li class="project-card bank">
+                <div class="text-container ">
+                    <h3 class="project-title">Profesional Art Printing Data</h3>
+                    <p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="list-container">
+                        <li class="project-list">html</li>
+                        <li class="project-list">bootstrap</li>
+                        <li class="project-list">ruby</li>
+                    </ul>
                 </div>
```

Comment:
This section should contain 6 card.three on top and three at the bottom. But your design only contains 3. 
This is your design: 
![Screenshot from 2022-05-31 18-23-13](https://user-images.githubusercontent.com/79506105/171201393-c4719bac-c629-4143-90ce-a0619a45a15e.png)

Kindly Follow this figma design please:
![Screenshot from 2022-05-31 18-29-25](https://user-images.githubusercontent.com/79506105/171201553-c13dd82f-d983-450e-b1c8-9b876e800da0.png)

Please carefully inspect the template and use the right [properties in your styling.

Reply:
done 
picture fixed as requested